src/Bundler/Pipeline/ContentPipeline.php

```diff
@@ -26,7 +26,7 @@
  * pushed, it will go through various processors until it is written to disk as
  * the given output file.
  */
-final class ContentPipeline implements ContentPipelineInterface
+class ContentPipeline implements ContentPipelineInterface
```

Comment:
final should remain here, shouldn't create an extension point for 3rd parties as they can extend it, which makes maintenance a lot harder if we would ever change something.

Solution would be to create a second interface and use this for the adders.